src/main/java/com/mraof/minestuck/item/weapon/OnHitEffect.java

```diff
@@ -314,6 +340,41 @@ static OnHitEffect enemyPotionEffect(Supplier<MobEffectInstance> effect)
 		return (stack, target, attacker) -> target.addEffect(effect.get());
 	}
 	
+	//Nearly identical to Sweep, but it applies an optional array of effects
+	static OnHitEffect SweepMultiEffect(OnHitEffect... effects)
```

Comment:
Make the first letter lowercase. We use the lowercase variant of camelCase in such conditions